### PR TITLE
p1-b1-r18 Paris

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-de4661/cod-bxxqqq_bHY6yh-de4661.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+              De iuramenti religione Rubrica 18
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e119">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de4661-Rd1e116">In leuitico capitulo 19</ref> <quote xml:id="bHY6yh-de4661-Qd1e125">non periurabis 
+              <lb ed="#B"/>in nomine meo nec pollues nomen 
+              <lb ed="#B"/>dei tui et cetera etc etc etc etc</quote>
+            </p>
+           <pb ed="#B" n="8-v"/>
+           <cb ed="#B" n="a"/>
+            <p xml:id="bHY6yh-de4661-d1e133">
```
Comment (jeffreycwitt) [inline bHY6yh-de4661/cod-bxxqqq_bHY6yh-de4661.xml:73]: page breaks and columns breaks always need to be inside a `<p>` tag. See my adjustment below.

```suggestion
            </p>
            <p xml:id="bHY6yh-de4661-d1e133">
              <pb ed="#B" n="8-v"/>
              <cb ed="#B" n="a"/>

```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
Looking good. Take a look at my note about the placement of page breaks and column breaks.

**Comment by GrantSchreib** [comment]
corrected

**Review by jeffreycwitt — APPROVED** [review]
looks good.

**Pull request merged**